test/index.js

```diff
@@ -1552,6 +1553,26 @@ See https://github.com/sanctuary-js/sanctuary-def/tree/v${version}#Array2 for in
     eq ($.Boolean.supertypes) ([]);
   });
 
+  test ('provides the "Buffer" type', () => {
+    eq ($.Buffer.name) ('Buffer');
+    eq ($.Buffer.url) (`https://github.com/sanctuary-js/sanctuary-def/tree/v${version}#Buffer`);
+    eq ($.Buffer.supertypes) ([]);
+
+    const isBuffer = $.test ([]) ($.Buffer);
+    eq (isBuffer (null)) (false);
+    eq (isBuffer (Buffer.from ([1, 2, 3]))) (true);
+
+    (Buffer => {
+      delete global.Buffer;
+      try {
+        eq (isBuffer (null)) (false);
+        eq (isBuffer (Buffer.from ([1, 2, 3]))) (false);
+      } finally {
+        global.Buffer = Buffer;
+      }
+    }) (Buffer);
```

Comment:
`delete global.Buffer` removes our ability to reference `Buffer` implicitly, so we must retain an explicit reference. `(Buffer => …) (Buffer)` strikes me as more elegant than `const Buffer_ = Buffer; …`.

Reply:
What about `const Buffer = global.Buffer`?

Reply:
Good idea. I did not consider `{ const Buffer = global.Buffer; … }`.